lib/src/components/NxTree/__tests__/NxTreeItemLabel.test.tsx

```diff
@@ -23,43 +23,57 @@ const keyNavContext: TreeKeyNavContextType = {
   setNavigationDirection: () => {},
   getTreeRoot: () => null
 };
+function getTreeItemLabel(extraProps?: Props) {
+  return (
+    <TreeKeyNavContext.Provider value={keyNavContext}>
+      <NxTree.ItemLabel { ...extraProps } />
+    </TreeKeyNavContext.Provider>
+  );
+}
+
+const quickRender = rtlRender(getTreeItemLabel, {}),
+    renderEl = rtlRenderElement(getTreeItemLabel, {});
 
 describe('NxTree.ItemLabel', function() {
-  it('makes a <span> tag with an nx-tree__item-label class', function() {
-    const component = mount(
-      <TreeKeyNavContext.Provider value={keyNavContext}>
-        <NxTreeItemLabel />
-      </TreeKeyNavContext.Provider>
-    ).children();
+  it('adds specified classes and attrs to the top-level element', function() {
```

Comment:
We can add a test for the top-level element as in the other test files as well.

Reply:
> We can add a test for the top-level element as in the other test files as well.

Added here: [e8ab884](https://github.com/sonatype/sonatype-react-shared-components/pull/950/commits/e8ab88409c1aded6b0c4241685f0d0fe4f927849). It's not testing for a role since `NxTree.ItemLabel` only has a generic role assigned to it